esp-hal/src/system.rs

```diff
@@ -5,15 +5,16 @@
 //! This `system` module defines the available radio peripherals and provides an
 //! interface to control and configure radio clocks.
 
+#[cfg(not(esp32p4))]
 use crate::peripherals::SYSTEM;
 
 /// Peripherals which can be enabled via `PeripheralClockControl`.
 ///
 /// This enum represents various hardware peripherals that can be enabled
 /// by the system's clock control. Depending on the target device, different
 /// peripherals will be available for enabling.
-// FIXME: This enum needs to be public because it's exposed via a bunch of traits, but it's not
-// useful to users.
+// NOTE: This enum needs to be public because it's exposed via a bunch of
```

Comment:
Please undo this, `FIXME` is actually highlighted if you use the Todo Tree vscode plugin.

Reply:
FIXME/TODO/NOTE are all pretty standard, plugin I use at least highlights all of those.

Reply:
Still, this change has no relevance for P4, and a FIXME better signals that something needs to be changed around these parts.

Reply:
will be done